api/tests/test.js

```diff
@@ -0,0 +1,94 @@
+'use strict'
+
+const os = require('os')
+const path = require('path')
+const fs = require('fs-extra')
+const nock = require('nock')
+const test = require('ava') //eslint-disable-line import/no-unresolved
+const request = require('supertest')
+const connection = require('../db/connection')
+const app = require('../index')
+const osmesa = require('./fixtures/user_json.json')
+const mostRecentlyActive = require('./fixtures/most_recently_active')
```

Comment:
it seems that most recently and least recently active users contain the same list. I recommend storing the user list as just a json file and the programmatically sort them in each test for validation.

Also please include `.json` in the require. It helps the reader realizes that this is a json file rather than a js module.